media/html/visjsDebuggerPanel.html

```diff
@@ -324,6 +361,49 @@
       function openAllClusters() {
         openClusterStack(Object.keys(network.body.nodes).filter((nodeId) => nodeId.startsWith('cluster_')));
       }
+
+      function showAllNodes() {
+        let nodeIds = Object.keys(network.body.nodes).filter((nodeId) => {
+          return !nodeId.startsWith('edge');
+        });
+
+        nodeIds.forEach((nodeId) => {
+          if (network.body.nodes[nodeId].options.hidden) {
+            setVisibility(nodeId, true);
+          }
+        });
+      }
+
+      function hideNode(event) {
+        const nodeId = network.getNodeAt({ x: event.pageX, y: event.pageY });
```

Comment:
```suggestion
        const nodeId = network?.getNodeAt({ x: event.pageX, y: event.pageY });
```
The button is already interactable before the network is created.